packages/palette/src/elements/Toggle/Toggle.tsx

```diff
@@ -57,21 +57,34 @@ export class Toggle extends React.Component<ToggleProps> {
 
   render() {
     const { disabled, expanded } = this.state
-    const { children, chevronSize, label, textSize, renderSecondaryAction } = this.props
+    const {
+      children,
+      chevronSize,
+      label,
+      textSize,
+      renderSecondaryAction,
+    } = this.props
+
+    const labelComponent =
```

Comment:
@damassi and @dzucconi should this also be moved inline/defined with title case?

Reply:
Ah, so, the rules are funny...

This is slightly different in that this is either a string (`label`) or straight up JSX (the `<Sans>` component); what makes a component a formal react component (per the convo in the other thread) is that its a function:

```tsx
const Label = props => {
  return <Sans />
}
```
To use the above, you have to invoke it:

```tsx
<Label />
```

(By 'invoke', it's helpful to think of <Label /> as equaling `Label()` -- invoking a function.) 

Wheres something like 
```tsx
const Label = isString ? label : <Sans>Hi</Sans> 
```

can't actually be invoked like 

```
<Label />
```
Because a) it could be a plain string; or b) if its not a string, then its just a reference to _already invoked_ JSX.

So in this case i think its fine to leave as is, and since it can be a string or component `labelComponent` is basically "label (string) or labelComponent (component)"; the naming emcompasses both possible forms.

Reply:
Ahhh yes okay really appreciate this explanation, _so_ helpful for solidifying my understanding of how React works under the hood.

Is this ready to merge? Does that storybook test need to be reviewed manually?

Reply:
Nope all good!